redis.go

```diff
@@ -1,12 +1,22 @@
 package redis // import "gopkg.in/redis.v3"
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net"
 	"time"
 )
 
+var (
```

Comment:
I don't think this is flexible enough, I like the approach of e.g. http://godoc.org/github.com/eaigner/jet#ComplexValue